lib/archethic/bootstrap.ex

```diff
@@ -207,13 +209,35 @@ defmodule Archethic.Bootstrap do
       Logger.info("Synchronization finished")
     end
 
+    persist(:oracle)
```

Comment:
Here you also have to persist all the genesis address.
You can add the function to persist oracle in the networkConstraint module, and add a flag in the function to determine if you are in bootstrap time or self repair time like `bootstrap?`
If you are on bootstrap you persist oracle, if not you don't